test-shared/src/main/java/org/odk/collect/testshared/FakeScheduler.kt

```diff
@@ -0,0 +1,65 @@
+package org.odk.collect.testshared
```

Comment:
Er annoyingly no dashes in package names. Maybe we just go with one word for module names then?

Reply:
I see. How about a `reused` module with `test` and `utility` packages or something like that? It would all be pure Java so should be no big deal to compile and I can't imagine there would be a ton shared anyway.

Reply:
Feels like a good idea to keep test and non-test code separate, so we're not bringing both in when we add a dependency to a module. Maybe for the moment we keep it as `testshared` and work out what we want to do when we have "shared" non-test code that doesn't feel like it needs its own module (like `async`). What do you think?

Reply:
Fair enough.